File: 3467_Transform_Array_by_Parity/solution.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int compareInts(const void* a, const void* b) { return (*(int*)a - *(int*)b); }
int* transformArray(int* nums, int numsSize, int* returnSize) {
    int* result = (int*)malloc(numsSize * sizeof(int));
    for (int i = 0; i < numsSize; i++) {
        if (nums[i] % 2 == 0) {
            result[i] = 0;
        } else {
            result[i] = 1;
        }
    }
    qsort(result, numsSize, sizeof(int), compareInts);
    *returnSize = numsSize;

    return result;
}

File: 3467_Transform_Array_by_Parity/solution.py
class Solution:
    def transformArray(self, nums: List[int]) -> List[int]:
        transformed_list = []
        for num in nums:
            if num % 2 == 0:
                transformed_list.append(0)
            else:
                transformed_list.append(1)
        transformed_list.sort()
        return transformed_list
